File: test/fixtures/sample-repo/src/breadth/greeter.ex
defmodule Greeter do
  def greet(name), do: format(name)
  def greet_safe(name) when is_binary(name), do: format(name)

  def run do
    greet("world")
  end

  defp format(name), do: String.upcase(name)
end

File: test/fixtures/sample-repo/src/breadth/greeter.zig
const Greeter = struct {
    pub fn greet(name: []const u8) []const u8 { return format(name); }
    fn format(name: []const u8) []const u8 { return name; }
};
pub fn run() []const u8 { return Greeter.greet("x"); }
